cli/command/stack/kubernetes/watcher.go

```diff
@@ -220,13 +220,11 @@ func newPodInformer(podsClient podListWatch, stackName string, indexers cache.In
 		&cache.ListWatch{
 			ListFunc: func(options metav1.ListOptions) (runtime.Object, error) {
 				options.LabelSelector = labels.SelectorForStack(stackName)
-				options.IncludeUninitialized = true
```

Comment:
Are these changes needed to make everything build? If so, perhaps better to include it in the same commit as the actual bump

Reply:
rebased and amended